### ISSUE TITLE
update WCAG info on our accessibility-info default content

### ISSUE THREAD

**Issue by tkimnguyen** [plone/Products.CMFPlone#3273]
https://demo.plone.org/en/accessibility-info currently says WCAG v2.0 twice. That should be updated to 2.1. See also https://github.com/plone/Products.CMFPlone/issues/2408

Note: https://www.w3.org/WAI/standards-guidelines/wcag/#versions "WCAG 2.2 is scheduled to be published in 2021"

**Comment by mohitbalwani** [comment]
Can I work on this issue?

**Comment by marwanatef2** [comment]
@tkimnguyen I think this issue should be closed? Or what do you think?

**Comment by tkimnguyen** [comment]
thank you @marwanatef2 !
